x-pack/plugins/security_solution/common/detection_engine/schemas/response/get_rule_execution_events_response.ts

```diff
@@ -8,10 +8,13 @@
 import * as t from 'io-ts';
 import { ruleExecutionEvent } from '../common';
 
-export const GetRuleExecutionEventsResponse = t.exact(
+export const GetRuleExecutionEventsResponse = t.intersection([
```

Comment:
If not deprecating `getLastFailures` this should move to its own dedicated type since we're now including additional information about total underlying executions.

Reply:
Types diverged and made a dedicated `GetAggregateRuleExecutionEventsResponse` type. Will still need to clean up this one if deprecating `getLastFailures`.